storybook/src/admin/Button.stories.tsx

```diff
@@ -75,14 +75,210 @@ export const Default = {
 export const AllVariants = {
     render: () => {
         return (
-            <Stack direction="row" spacing={2}>
-                <Button variant="primary">Primary</Button>
-                <Button variant="secondary">Secondary</Button>
-                <Button variant="outlined">Outlined</Button>
-                <Button variant="destructive">Destructive</Button>
-                <Button variant="success">Success</Button>
-                <Button variant="textLight">Text Light</Button>
-                <Button variant="textDark">Text Dark</Button>
+            <Stack spacing={10}>
+                <Stack direction="row" spacing={8}>
```

Comment:
Could you please add `alignItems="center"` to each of the rows? 

Since the usage of `Chip` is currently a bit broken the height increaes in all buttons, with `alignItems="center"` only the "broken" buttons would have the incorrect height. 

_Chip-Support will be added in [COM-1600](https://vivid-planet.atlassian.net/browse/COM-1600) btw._